spec/validators/presence_partner_optional_validator_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+class DummyPartnerPresenceClass
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :val1, :val2, :val3
+
+  validates :val1, presence: true
+  validates :val2, presence_partner_optional: { partner_labels: :partner_check_true? }
```

Comment:
The validator requires partner_labels to be a method it can call.  If we send 
```ruby
validates :val2, presence_partner_optional: { partner_labels: true }
```
The validator will try and call object.send(true) and fail.
The value has to be processed by the record being validated so these tests have to replicate it 🤷‍♂️ 
Happy to chat if you meant something different?